optbinning/scorecard/plots.py

```diff
@@ -0,0 +1,289 @@
+"""
+Modules with plots for visualizing model performance.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.metrics import roc_curve, roc_auc_score
+
+
+def plot_auroc(
```

Comment:
- General comment: to be consistent with other classes, replace function arguments by:

```python

def plot_auc_roc(df, target_col, proba_col, title="ROC curve", xlabel="False positive rate",
                 ylabel="True positive rate", roc_line_color="blue", reference_line_color="black",
                 savefig=False, figname=None):

```
- Changes:
   - Function name: plot_auc_roc or plot_roc_auc. See: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#sklearn.metrics.roc_auc_score
   - Add line to color parameters.
   - figname is None by default, the user must provide a name.

Reply:
Couldn't understand the "Add line to color parameters"